### PR TITLE
Used loggers in place of print in sysinfo.py

### PR DESCRIPTION
**Pull request by agneet42**
Used logger.info for printing out information instead of  print() function

### CODE REVIEW

**Comment by agneet42** [comment]
@dpshelio @Cadair , have a look please.

File: sunpy/util/sysinfo.py [hunk 1]
```diff
@@ -97,33 +101,39 @@ def system_info():
...
+    
 
     for sys_info in ['Time', 'System', 'Processor', 'Arch', 'SunPy', 'SunPy_git']:
-        print('{0} : {1}'.format(sys_info, sys_prop[sys_info]))
+        logger.info('%s : %s', sys_info, sys_prop[sys_info])
 
     if sys_prop['System'] == "Linux":
         distro = " ".join(platform.linux_distribution())
-        print("OS: {0} (Linux {1} {2})".format(distro, platform.release(), sys_prop['Processor']))
+        logger.info("OS: %s (Linux %s %s), distro, platform.release(), sys_prop['Processor'])
```
Comment (nitinkgp23) [inline sunpy/util/sysinfo.py]: You have a syntax error here.

Comment (agneet42) [inline sunpy/util/sysinfo.py]: I'll make the change. Thanks.

**Comment by Cadair** [comment]
The purpose of `sysinfo()` is to print information out to the user to give as a debugging tool in issues or mailing list posts. What has motivated you to change it to using logger?

**Comment by agneet42** [comment]
@Cadair ( https://github.com/sunpy/sunpy/issues/1859 ) explicitly mentioned a set of files which uses print statements. I used logger here to print out the info and debugging errors with labels.

If there are files, which actually need logger, could you specify the same?

**Comment by Cadair** [comment]
I will defer to @dpshelio as he opened the initial issue.

**Pull request closed without merging**